Source/carthage/Build.swift

```diff
@@ -303,7 +303,9 @@ extension BuildPlatform: ArgumentType {
 	]
 
 	public static func fromString(string: String) -> BuildPlatform? {
-		let commaSeparated = string.characters.split(allowEmptySlices: false) { $0 == "," }.map(String.init)
+		let tokens = string.characters
+			.split(allowEmptySlices: false) { $0 == "," || $0 == " " }
+			.map(String.init)
 
 		let findBuildPlatform: String -> BuildPlatform? = { string in
 			for (key, platform) in self.acceptedStrings {
```

Comment:
This could be
```swift
return lazy(self.acceptedStrings)
    .filter { key, _ in string.caseInsensitiveCompare(key) }
    .map { _, platform return platform }
    .first
```